### DISCUSSION TITLE
Any chance of this eventually getting merged into trino?

### DISCUSSION THREAD

**Discussion by lozbrown** [ebyhr/trino-storage#321] [General]
This would be really useful for ETL. 

As with all these things patching this into the container images for every version could be quite an overhead when trying to maintain versions.

**Comment by nineinchnick** [comment]
A while ago, we added polymorphic table functions, instead of only relying on the hack where we use a URL as a table name. This is great, because we can use more parameters in these functions, like specify some format options (CSV separators, etc.). If we'd consider contributing this to Trino, most of the work would come to agreeing on the naming conventions for these functions, and their arguments. We can start the discussion here, and after reaching some level of confidence, work on a PR in Trino. As usual, we're open to contributions.
